ghost/admin/app/routes/home.js

```diff
@@ -9,6 +9,11 @@ export default class HomeRoute extends Route {
     beforeModel(transition) {
         super.beforeModel(...arguments);
 
+        const toolbarJSON = new URLSearchParams(window.location.hash.substring(1)).get('__posthog');
+        if (toolbarJSON) {
+            window.posthog?.loadToolbar(JSON.parse(toolbarJSON));
```

Comment:
Are we okay doing calls to `window.posthog` (which is loaded in `pro.js`) in Admin?